index.js

```diff
@@ -179,6 +179,10 @@ function checkRootPathForErrors (rootPath) {
   try {
     pathStat = statSync(rootPath)
   } catch (e) {
+    if (e.code === 'ENOENT') {
+      return new Error('"root" should point to an existing directory')
```

Comment:
Can you keep the code `ENOENT`  attached to the error? Also, can you include `rootPath` in the error message?

Reply:
Sure, @mcollina, just wondering do you mean change "root" to "rootPath", or the value of root path?

Reply:
I meant the value of `rootPath`.